src/pages/register/index.tsx

```diff
@@ -0,0 +1,96 @@
+import styled from '@emotion/styled'
+import { TextInput } from '@components/basic/InputBox'
+import { FillBtn } from '@components/basic/button/Button'
+import { ChangeEvent, useState, useEffect } from 'react'
+import { useRouter } from 'next/router'
+import { request } from '@utils/api'
+import { PutStudentIDParams, PutStudentIDResults } from '@api/insertStudentID'
+import { useDispatch, useSelector } from 'react-redux'
+import { RootState } from '@redux/store'
+import { addStudentID } from '@redux/features/userSlice'
+import { MOBILE_WIDTH_THRESHOLD } from 'src/constants/ui'
+import { palette, typography } from '@styles/theme'
+
+export default function StudentID() {
+  const studentID = useSelector((state: RootState) => state.userInfo.studentID)
+  const [insertedStudentID, setInsertedStudentID] = useState('')
+  const dispatch = useDispatch()
+  const { push } = useRouter()
+
+  useEffect(() => {
```

Comment:
let's remove this automatic routing. Students might need to update their student Id after initial registration?